rskj-core/src/main/java/org/ethereum/rpc/Web3.java

```diff
@@ -198,6 +198,10 @@ public String toString() {
     boolean personal_lockAccount(String key);
     String personal_dumpRawKey(String address) throws Exception;
 
+    String txpool_content();
```

Comment:
Maybe we can have something like this
```java
public interface TxPoolWeb3Module {

    default String txpool_content() {
        return getModule().content();
    }

    default String txpool_inspect() {
        return getModule().inspect();
    }

    default String txpool_status() {
        return getModule().inspect();
    }

    TxPoolModule getModule();
}
```
and make `org.ethereum.rpc.Web3Impl` to implement it. Then this class should only really implement the `getModule()` returning the injected _TxPoolModuleImpl_ field and we let the actual forwarding to happen inside the default methods of the interface

Reply:
Interesting, but it makes me wonder whether that will work with `jsonrpc4j` mapping.
At the same time, this looks like a hack for our contrived architecture, but i'm not against it :)

Reply:
Also, the `getModule` name would clash with others later on. How about:

```java
public interface Web3 {

    default String txpool_content() {
        return getTxPoolModule().content();
    }

    default String txpool_inspect() {
        return getTxPoolModule().inspect();
    }

    default String txpool_status() {
        return getTxPoolModule().inspect();
    }

    TxPoolModule getTxPoolModule();

    public interface TxPoolModule {

        String content();

        String inspect();

        String status();
    }
}
```

My concern is that if we followed this pattern for all our modules, the `Web3Impl` class would do nothing. What would be the responsibility of this class?:

```java
public class Web3Impl implements Web3 {
    public Web3Impl(Module1 m1, Module2 m2, ...) {
        this.m1 = m1;
        this.m2 = m2;
        ...
    }

    public Module1 M1 getModule1() {
        return this.m1;
    }

    public Module2 M2 getModule2() {
        return this.m2;
    }
    ...
}
```

Reply:
Yes, I edited the `getModule` method right after sending it :)
I think jsonrpc4j should still work b/c it uses _reflection_ to resolve the method of `Web3Impl` to execute, but of course that will be the first test to run.
The only responsibility of this class will be to be useful in the context of jsonrpc4j, just that.

Reply:
I like the idea about an interface with a default implementation that web3 require for modules. I don't think it's a good idea to tie both interfaces. I will add default implementation for txpool module. We can in other PR introduce them for others modules